modules/install-osp-dualstack.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-installer-custom.adoc
+:_content-type: CONCEPT
+[id="install-osp-dualstack_{context}"]
+= Optional: Configuring a cluster with dual-stack networking
+
+:FeatureName: Dual-stack configuration for OpenStack
+include::snippets/technology-preview.adoc[]
+
+You can create a dual-stack cluster on {rh-openstack}. However, this is enabled only if you are using an {rh-openstack} network with IPv4 and IPv6 subnets. To configure a dual-stack cluster, you must complete the following tasks:
+
```

Comment:
Change all introductory phrases in list items to the imperative voice (I've suggested the changes below) rather than using gerund phrases in sentence fragments. The phrase `you must do the following tasks:` that introduces the list implies that a set of actions that the user must take will follow, and changing to the imperative creates a clear set of actions.